crates/sui-core/src/consensus_types/consensus_output_api.rs

```diff
@@ -109,7 +109,8 @@ impl ConsensusOutputAPI for consensus_core::CommittedSubDag {
             Some(
```

Comment:
The API here assumes that the scores are sorted in scoring `desc` (and consumers do use them as such) . So let's sort them before returning